scripts/update_embedded.sh

```diff
@@ -60,5 +60,6 @@ performUpdate() {
 	rm -f "$TEMP_ETAG_FILENAME"
 }
 
+# The following URLs shall match the ones in AppURLs.swift. Danger checks that the URLs match on every PR. If the code changes, the regex that Danger uses may need an update.
 performUpdate 'https://staticcdn.duckduckgo.com/trackerblocking/v5/current/ios-tds.json' "${base_dir}/Core/AppTrackerDataSetProvider.swift" "${base_dir}/Core/trackerData.json"
 performUpdate 'https://staticcdn.duckduckgo.com/trackerblocking/config/v2/ios-config.json' "${base_dir}/Core/AppPrivacyConfigurationDataProvider.swift" "${base_dir}/Core/ios-config.json"
```

Comment:
Whoa, at first I thought that the Danger error for this PR was a false negative, but this URL actually _is_ different than the one in `AppURLs.swift` and should be updated 😳 At least we've just verified that your new Danger check works :)

Reply:
Cool! 
I've updated the link and embedded files.